Google.PowerShell/Logging/GcLogCmdlet.cs

```diff
@@ -640,4 +640,53 @@ protected override void ProcessRecord()
             WriteLogEntriesResponse response = request.Execute();
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Remove logs.
```

Comment:
While this is probably sufficient for a programmer, we should be more verbose for user-facing documentation. "Remove (how many?) logs from the provided Cloud Logging (group? bucket? machine?)